dsc_lib/src/configure/config_result.rs

```diff
@@ -102,6 +147,27 @@ pub struct ResourceTestResult {
     pub result: TestResult,
 }
 
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
```

Comment:
Is there a strong reason to deny_unknown_fields ?

Reply:
We can evaluate if we want to allow arbitrary fields later if a scenario comes up